src/components/MenuOpt.tsx

```diff
@@ -0,0 +1,33 @@
+import { MenuOptProps } from '../type/types'
+import { NavLink } from 'react-router-dom'
+
+export default function MenuOpt({ children, to, closeMenu, isLogout }: MenuOptProps) {
+  if (isLogout) {
+    return (
+      <NavLink
+        to={to}
+        onClick={() => {
+          localStorage.removeItem('token')
+          localStorage.removeItem('userId')
+          closeMenu && closeMenu()
+        }}
+        className="flex w-full items-center gap-3 rounded-lg px-4 py-3 text-left text-white transition-colors hover:bg-slate-600"
+      >
+        {children}
+      </NavLink>
+    )
+  }
+  return (
+    <NavLink
+      to={to}
+      onClick={() => {
+        closeMenu && closeMenu()
+      }}
```

Comment:
This syntax could have been more simplified and readable
```suggestion
      onClick={closeMenu}
```